app/page/template/partials/template-message.htm

```diff
@@ -231,14 +228,6 @@
           </div>
         <!-- /ko -->
       </div>
-      <div class="message-body" data-bind="foreach: remoteUserEntities">
```

Comment:
Do we still use remoteUserEntities elsewhere or can we remove it from member messages.

Reply:
ConversationCellState still relies on it.